packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md

```diff
@@ -16,70 +16,81 @@ This guide explains how to customize the Sign-In Widget when Okta is hosting it
 
-1. Make changes directly in the embedded editor.
-2. Click **Preview** to preview your changes before you publish.
-3. Click **Reset to Default** if you need to remove all of your customizations and restore the default HTML/CSS and JavaScript code.
-4. Click **Save and Publish** when you finish.
+Triple curly braces (`{{{`) are only used for the `errorDescription` variable to return unescaped HTML.
 
-> **Note:** See the [Customization examples](#customization-examples) section for examples that you can alter and use on your hosted sign-in page.
+| Variable | Contains |
+|----------|-------------|
+| <span v-pre>`{{orgName}}`</span> | The org name title |
+| <span v-pre>`{{errorSummary}}`</span> | The error title text |
+| <span v-pre>`{{bgImageUrl}}`</span> | The URL to the background image configured for your application. You can change this image by using the [Sign-in Configuration](https://help.okta.com/okta_help.htm?type=oie&id=ext-branding-set-theme) option, but this changes the background image in all instances where the variable is used, including your custom sign-in page. If you want to change only the background image for your custom error pages, include the URL to the image instead of the variable. |
+| <span v-pre>`{{orgLogo}}`</span> | The logo image that has been configured for your application. You can change this logo by using the [Sign-in Configuration](https://help.okta.com/okta_help.htm?type=oie&id=ext-branding-set-theme) option, but this changes the org logo in all instances where the variable is used, including your custom sign-in page. If you want to change only the logo image for your custom error pages, include the URL to the image instead of the variable. |
+| <span v-pre>`{{{errorDescription}}}`</span> | A detailed description of the error |
+| <span v-pre>`{{back}}`</span> | The text "Go to Homepage". When the user clicks the button, they are returned to the sign-in page. |
+| <span v-pre>`{{technicalDetails}}`</span> | Any additional messaging, if the error code has any. Here are sample technical details for an error code:</br>"If you are using custom expressions like `\{0}`, make sure that the field `customField` is present in the user profile. Please review your attribute list and make the appropriate change."</br>See [Okta Error Codes](/docs/reference/error-codes/#okta-error-codes-listed-by-error-code). |
+| <span v-pre>`{{buttonText}}`</span> | Inserts the button text based on the page context. When the user selects the button, they are directed to the `buttonHref` URL. The <span v-pre>`{{back}}`</span> variable is also supported for the same purpose. |
+| <span v-pre>`{{buttonHref}}`</span> | The hyperlink for the button |
+| <span v-pre>`{{themedStylesUrl}}`</span> | The URL for the themed style sheet |
+| <span v-pre>`{{faviconUrl}}`</span> | The URL for the favicon |
```

Comment:
<img width="459" alt="Screen Shot 2022-12-02 at 9 47 21 AM" src="https://user-images.githubusercontent.com/49454026/205354062-b28b47b7-85db-467a-a60e-92990170829c.png">
these are the only compatible variables with sign in page

Reply:
the variables compatiuble with sign in page are already called out below in "Use macros and request context", so we should just do the table or that, but not both

Reply:
removed the table and moved the intro content to the macros section. also swapped "macros" for "variables"